arwen/cryptoapi/cryptoei.go

```diff
@@ -980,12 +1458,54 @@ func v1_4_unmarshalCompressedEC(
 		return int32(len(data))
 	}
 
+	return commonUnmarshalCompressedEC(context, xResultHandle, yResultHandle, ecHandle, data)
```

Comment:
Can you confirm this change is backwards compatible? (i.e., consensus will work on this kind of errors, before the new features get activated).

Reply:
yes. it is.